tests/DMSDocumentCartCheckoutPageTest.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+class DMSDocumentCartCheckoutPageTest extends FunctionalTest
+{
+    protected static $fixture_file = 'DMSDocumentCartTest.yml';
+    /**
+     * @var DMSDocumentCartCheckoutPage_Controller
+     */
+    protected $controller;
+
+    /**
+     * @var DMSDocumentCart
+     */
+    protected $cart;
+
+    public function setUp()
+    {
+        parent::setUp();
+        $this->controller = DMSDocumentCartCheckoutPage_Controller::create();
+        $this->cart = $this->controller->getCart();
+        DMSDocumentCartCheckoutPage_Controller::add_extension('StubDMSDocumentCheckoutPageExtension');
+        Injector::inst()->registerService(new StubEmail(), 'Email');
+    }
+
+    /**
+     * Tests DMSDocumentRequest form has a FieldList
+     */
+    public function testDMSDocumentRequestForm()
+    {
+        $form = $this->controller->DMSDocumentRequestForm();
+        $this->assertInstanceOf('FieldList', $form->Fields());
+    }
+
+    /**
+     * Tests if the DMSDocumentRequestForm is extensible
+     */
+    public function testDMSDocumentRequestFormIsExtensible()
+    {
+        $controller = DMSDocumentCartCheckoutPage_Controller::create();
```

Comment:
This was a remnant of a previous PR change (moving adding of the stub extension to setUp). I have updated it.